app/views/media_objects/_file_upload.html.erb

```diff
@@ -112,6 +115,7 @@ Unless required by applicable law or agreed to in writing, software distributed
                 <% end %>
               </div>
             </div>
+            -->
```

Comment:
@jasoncorum which part of ☝️ was failing when our ingested media object is missing duration? And how was it failing? I'm concerned that commenting out the HTML will not prevent a Ruby error.

Reply:
@afred - this is a bad comment. i commented this field out of the view because we all discussed it a few weeks back and we decided that we'd rather force the user to select the still via the player versus entering a specific timecode in the form here. 

when i manually tested what would happen if a user entered a timecode beyond the actual duration of the media, the job handling the thumbnail creation would eventually fail and the thumbnail just wouldn't get updated.

i'll make a better comment and resubmit.

Reply:
@jasoncorum ah... gotcha. I didn't see at first that it was a form field. My concern was that if `poster_offset` was undefined, or if `master_files[#{part.id}]` was nil, or if `part.poster_offset` were nil, then we'd end up with ruby errors that the `<!-- -->` comment would not prevent.

That being said... if we're committing to a change in teh UI as to how the offset is selected, I'm fine with just deleting the whole form element altogether.